source/time.tex

```diff
@@ -9505,8 +9508,8 @@
 \pnum
 \effects
 Constructs a \tcode{zoned_time} by
-initializing \tcode{zone_} with \tcode{std::move(z)}.
-% FIXME: What about tp_?
+initializing \tcode{zone_} with \tcode{std::move(z)} and
+default constructing tcode{tp_}.
```

Comment:
```suggestion
default constructing \tcode{tp_}.
```
Following the first change in this file, the first line of the `\effects` should be dropped and "initializing" changed to "Initializes".

Reply:
It should be, but let's address that separately and more holistically.

Reply:
Filed #3506 to track.